pkg/local_object_storage/metabase/revive.go

```diff
@@ -0,0 +1,155 @@
+package meta
+
+import (
+	"fmt"
+
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/util/logicerr"
+	"github.com/nspcc-dev/neofs-sdk-go/object"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+	"go.etcd.io/bbolt"
+)
+
+// ErrObjectWasNotRemoved is returned when object neither in the graveyard nor was marked with GC mark.
+var ErrObjectWasNotRemoved = logicerr.New("object neither in the graveyard nor was marked with GC mark")
+
+// ErrReviveFromContainerGarbage is returned when the object is in the container that marked with GC mark.
+var ErrReviveFromContainerGarbage = logicerr.New("revive from container marked with GC")
+
+type reviveStatusType int
+
+const (
+	// ReviveStatusGraveyard is the type of revival status of an object from a graveyard.
+	ReviveStatusGraveyard reviveStatusType = iota
+	// ReviveStatusGC is the type of revival status of an object from GC.
```

Comment:
"from GC" confuses me a little. "from the garbage bucket"? of explicitly say that GC mark was removed